security-policies/bundle/compliance/cis_azure/rules/cis_2_1_20/test.rego

```diff
@@ -0,0 +1,91 @@
+package compliance.cis_azure.rules.cis_2_1_20
+
+import data.cis_azure.test_data
+import data.compliance.policy.azure.data_adapter
+import data.lib.test
+import future.keywords.if
+
+test_violation if {
+	eval_fail with input as test_data.generate_security_contacts([test_data.generate_single_security_contact(
+		"default",
+		prop_alert_notifications({
+			"state": "Off",
+			"minimalSeverity": "Medium",
+		}),
+	)])
+
+	eval_fail with input as test_data.generate_security_contacts([test_data.generate_single_security_contact(
+		"default",
+		prop_alert_notifications({"state": "On"}),
+	)])
+
+	eval_fail with input as test_data.generate_security_contacts([test_data.generate_single_security_contact(
+		"non-default",
+		prop_alert_notifications({
+			"state": "On",
+			"minimalSeverity": "Medium",
+		}),
+	)])
+
+	eval_fail with input as test_data.generate_security_contacts([
+		test_data.generate_single_security_contact(
+			"non-default",
+			prop_alert_notifications({
+				"state": "On",
+				"minimalSeverity": "High",
+			}),
+		),
+		test_data.generate_single_security_contact(
+			"default",
+			prop_alert_notifications({
+				"state": "On",
+				"minimalSeverity": "Medium",
+			}),
+		),
```

Comment:
why would a lower severity alert trigger than the recommended one will fail the rule?

Reply:
Yes, you are right; perhaps it's better to accept both `medium` and `low` settings since they will also cover `high` level alerts.

Reply:
Hmm, then this means we only need to check the `state` to be `on` since whatever option is selected in `minimumSeverity` will include `high` as a minimum.

Reply:
I agree "Ensure that the Notify about alerts with the following severity **(or higher)**: setting is checked and set to High"